extensions/wikia/PortableInfobox/services/Helpers/ImageFilenameSanitizer.php

```diff
@@ -44,15 +48,45 @@ private function getFilePrefixRegex( $contLang ) {
 	 * @return mixed
 	 */
 	public function sanitizeImageFileName( $filename, $contLang ) {
-		// replace the MW square brackets and surrounding whitespace
-		$trimmedFilename = trim( $filename, "\t\n\r[]" );
-
+		$plainText = $this->convertToPlainText( $filename );
 		$filePrefixRegex = $this->getFilePrefixRegex( $contLang );
+		$textLines = explode( PHP_EOL, $plainText );
```

Comment:
As far as I know, yes. If you have a counterexample, I'm more than willing to factor it in. Aside, this is an additional part of functionality, so we can iterate if the assumption is wrong.